samples/sensor/bmg160/README.rst

```diff
@@ -0,0 +1,31 @@
+.. zephyr:code-sample:: bmg160
+   :name: BMG160 3-axis gyroscope
+   :relevant-api: sensor_interface
+
+   Get temperature, and angular velocity data from a BMG160 sensor.
+
+Overview
+********
+
+This sample shows how to use the Zephyr :ref:`sensor` API driver for the
+`Bosch BMG160`_ environmental sensor.
+
+.. _Bosch BMG160:
+   https://www.mouser.com/datasheet/2/783/BST-BMG160-DS000-1509613.pdf
```

Comment:
Can you link to the datasheet on bosch.com?

Reply:
I did look for it but couldn't find it on their website. I think this sensor isn't the only one where it's the case (there are other datasheets from mouser in the repo which I guess were used for this same reason)

Reply:
I managed to found an archived link from Bosch website